### ISSUE TITLE
Activating the plugin instantly crashes the Godot editor.

### ISSUE THREAD

**Issue by timvancann** [Atlinx/Godot-Mono-CustomResourceRegistry#4]
I'm not sure why it happens but if I activate the plugin the Godot editor instantly crashes. I'm on Mac running Godot_mono v3.3.

**Comment by Atlinx** [comment]
Were you able to get any errors from the console output? This may be a platform-specific bug since can't seem to reproduce this on my windows computer.

**Comment by Atlinx** [comment]
I've tested this on Mac Sur 11.3.1 with the latest version of Godot Mono and the plugin works. Have you tried building the plugin once before enabling it?

**Comment by timvancann** [comment]
Hi, thanks for getting back to me. I did all your steps and it still crashed, but I also noticed that LINQ expressions were not being detected in my IDE so I messed around with the TargetFramework in the solution settings and changed it to `netstandard2.1`. After that everything started working and the custom resources are now being detected!

I'm not sure why the default of `net472` doesn't work, even though I have all the latest frameworks. But that is unrelated to this library :).
